examples/text-replacer/background.js

```diff
@@ -0,0 +1,29 @@
+chrome.commands.onCommand.addListener((command, tab) => {
+  if (command == 'replace-text') {
+    replaceText(tab.id);
+  }
+});
+
+chrome.runtime.onInstalled.addListener(() => {
+  registerContextMenus();
+});
```

Comment:
During extension update, will this cause an error by trying to re-register context menu items?